txmongo/_gridfs/__init__.py

```diff
@@ -138,6 +141,45 @@ def get(self, file_id):
         defer.returnValue(GridOut(self.__collection, doc))
 
     @defer.inlineCallbacks
+    def get_version(self, filename=None, version=-1, **kwargs):
```

Comment:
If you have decided to omit custom filtering feature, lets drop `**kwargs` here too in order to not make an illusion that it will work as in PyMongo